java/Alena-Vorokh/src/com/zoolatech/lecture1/tasks/_7/Task_7a.java

```diff
@@ -0,0 +1,23 @@
+package com.zoolatech.lecture1.tasks._7;
+
+/* Write a program that prints all odd numbers from N to 1. N should be read from the input.
+Every number should be printed on the new line. First version of the program:
+use the for loop.*/
+
+import java.util.Scanner;
+
+public class Task_7a {
+    public static void main(String[] args) {
+    Scanner scanner = new Scanner(System.in);
+    System.out.println("Input Number: ");
+    int N = scanner.nextInt();
```

Comment:
same: please use variable names that start from a lowercased character: `int N` -> `int n`